wca-guidelines.md

```diff
@@ -106,6 +106,7 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-7><environment><environment> Article 7: Environment
 
 - 7f1a+) [CLARIFICATION] A full-size mat should have minimum dimensions of 30cm (left to right) by 25cm (front to back).
+- 7f2+) [CLARIFICATION] Attempts done with an unattached timer may not be replaced or disqualified if it went unnoticed when the solve was finished and the competitor gained no advantage from it.
```

Comment:
This seems a bit weird to me, since it's phrased carefully to suggest that they may be replaced *if* there was cheating. Can we do something like "even if" or "Exception:"?

Reply:
Do you have a suggestion for the wording?

Reply:
I think it might be better to withhold from adding this to the next Regs, but perhaps this?
```suggestion
- 7f2+) [CLARIFICATION] Attempts done with an unattached timer may not be replaced or disqualified if it went unnoticed when the solve was finished even if the competitor gained no advantage from it).
```